editoast/src/views/timetable/paced_train.rs

```
@@ -387,15 +385,14 @@ async fn get_path(
         PacedTrainError::NotFound { paced_train_id }
     })
     .await?;
+
+    let train_schedule = exception_key
+        .and_then(|key| paced_train.exceptions.iter().find(|e| e.key == key))
+        .map(|exception| paced_train.apply_exception(exception))
+        .unwrap_or_else(|| paced_train.into_first_occurrence());
```

What happens if the `exception_key` is invalid? I believe this code will fallback to the base occurrence, but I would argue that we probably want to return an error to the user instead.